src/main/java/core/basesyntax/readdata/DataReadingImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.readdata;
+
+import core.basesyntax.DataBase;
+import core.basesyntax.FruitTransaction;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class DataReadingImpl implements DataReading {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public void readData(String fileName) {
```

Comment:
ми повинні повертати результат з методів, щоб можна було потім їх тестувати, і загалом це вважається хорошою практикою
```suggestion
    public List<String>/String[]/... readData(String fileName) {
```